.circleci/config.yml

```diff
@@ -217,20 +232,44 @@ jobs:
     working_directory: /go/src/github.com/kubeapps/kubeapps
     environment:
       GOPATH: /home/circleci/.go_workspace
-      IMAGES: "kubeapps/apprepository-controller kubeapps/tiller-proxy"
+      IMAGES: "apprepository-controller tiller-proxy"
     <<: *build_images
   build_dashboard:
     docker:
       - image: circleci/golang:1.11
     environment:
-      IMAGES: "kubeapps/dashboard"
+      IMAGES: "dashboard"
     <<: *build_images
   release:
     docker:
       - image: circleci/golang:1.11
     steps:
       - checkout
       - run: REPO_DOMAIN=kubeapps REPO_NAME=kubeapps ./script/create_release.sh ${CIRCLE_TAG}
+  local_e2e_tests:
+    working_directory: /home/circleci/.go_workspace/src/github.com/kubeapps/kubeapps
```

Comment:
why is this necessary?

Reply:
it's probably not, we are not compiling in this job